src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+  height: 16px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--4 .stars__star {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(n + 5) {
```

Comment:
Use nesting as you work with `scss`, and add styles for `stars--1`, `stars--2`, etc, in case of any changes